backbone.js

```diff
@@ -79,6 +79,9 @@
         return _[method](this[attribute], value);
       };
       case 3: return function(iteratee, context) {
+        if (predicateTransform && !_.isFunction(iteratee) && _.isObject(iteratee)) {
```

Comment:
I'm not sure we should limit to just `{ attr: 'value' }` iterating syntax. Think, `col.find('active')`, etc.